packages/integration-tests/index.js

```diff
@@ -502,24 +506,49 @@ async function cleanupProviders(stackName, bucket, providersDirectory, postfix)
  *
  * @param {string} config - Test config used to set environment variables and template rules data
  * @param {string} dataDirectory - the directory of rules json files
- * @param {string} overrides - override rule fields
+ * @param {Object} overrides - override rule fields
+ * @param {string} postfix - string to append to rule name, collection, and provider
  * @returns {Promise.<Array>} array of Rules added
  */
-async function addRules(config, dataDirectory, overrides) {
+async function addRulesWithPostfix(config, dataDirectory, overrides, postfix) {
```

Comment:
How about "suffix"?

Reply:
So sure, but whenever we're appending to collections, providers, etc in integration tests it's called `postfix`, so maybe not the best term but consistent with the others? what do you think?

Reply:
Since it's tests and not code the public is going to use, it isn't a hill I'm going to die on.  If `postfix` is consistent, then that's fine.